.flake8

```diff
@@ -2,3 +2,6 @@
 ignore = E203, E266, W503, E402
 max-line-length = 88
 exclude = tests/yatai/proto, proto, _version.py
+per-file-ignores =
+    docker/manager.py:F405, F403, E501
```

Comment:
Any reason to ignore these flags?

Reply:
I will keep E501 for now (line-too-long). Before i use star imports